ui/templates/upload.html

```diff
@@ -1,5 +1,6 @@
-{% extends "base.html" %}
+{% extends "repo_base.html" %}
```

Comment:
Should this extend repo_base or just base?

I'm seeing the import button from repo_base, which we don't want here. 

![screen shot 2015-06-03 at 4 28 09 pm](https://cloud.githubusercontent.com/assets/430126/7970822/99502434-0a0e-11e5-91ab-71a16cca7862.png)